src/scrape.py

```diff
@@ -1,35 +1,58 @@
 import pandas as pd
 import util.scrape_util as scrape_util
 import constants
+import time
+
+start_time = time.time()
+
+CSV_PATH = constants.BASE_DIR + "src/address_with_days1.csv"
 
 def run():
     """
     Step 1) Import address csv.
     Step 2) Choose address.
     Step 3) Query addresss url for trash days.
+
+    Returns trash_df : pandas df with "address" and "trash_day" columns.
+    Addresses with multiple trash days are split into a new line for each day.
     """
-    trash_days_list = []
+    trash_days_dict = {}
     test_flag = False
+    driver = scrape_util.initChromeDriver()
 
     if test_flag:
         trash_days_dict = {}
         test_address = "160 15th Street, Brooklyn, NY, USA"
-        trash_days = scrape_util.getTrashDays(test_address, screenshot=True)
+        trash_days = scrape_util.getTrashDays(test_address, driver, screenshot=False)
         trash_days_dict[test_address] = trash_days
-        trash_days_list.append(trash_days_dict)
 
     else:
         address_list_path = constants.BASE_DIR + "src/Addresses11215.csv"
-        address_list = pd.read_csv(address_list_path)[0:3]
+        address_list = pd.read_csv(address_list_path)[:10]
         print("searching %s addresses" % len(address_list.index))
 
         for index, row in address_list.iterrows():
-            trash_days_dict = {}
             address = str(row["Address"]) + ", Brooklyn, NY, USA"
-            print("checking address %s" % str(index+1))
-            trash_days = scrape_util.getTrashDays(address, screenshot=False)
+            print("checking address %s of %s" % (str(index+1) , len(address_list)))
+            print("program has been running for %s seconds" % str(time.time() - start_time))
+            trash_days = scrape_util.getTrashDays(address, driver, screenshot=False)
             trash_days_dict[address] = trash_days
-            trash_days_list.append(trash_days_dict)
+        
+        ## clear addresses with NoneType trash_days
+        clean_dict = {k: v for k, v in trash_days_dict.iteritems() if v is not None} 
```

Comment:
why would we have a NoneType trash day? If we do, this should be functionalized and then called inside the getTrashDays

Reply:
This was the case for trash days that failed to scrape (i.e. if the address doesn't have any associated trash day, or if the browser fails to load the page). But, again, scrape.py should be ignored.